### PR TITLE
[PC-2049] Fix swift startWithApiKey instances

### PR DESCRIPTION
**Pull request by Bucimis**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/push_story.md [hunk 1]
```diff
@@ -237,7 +237,7 @@ appboyOptions[ABKPushStoryAppGroupKey] = @"YOUR_APP_GROUP";
 let appboyOptions: [AnyHashable: Any] = [
   ABKPushStoryAppGroupKey : "YOUR_APP_GROUP"
 ]
-Appboy.start(withApiKey: "YOUR_APPBOY_API_KEY", in:application, withLaunchOptions:launchOptions, withAppboyOptions: appboyOptions)
+Appboy.start(withApiKey: "YOUR_APPBOY_API_KEY", in:application, withLaunchOptions:launchOptions, withAppboyOptions:appboyOptions)
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/push_story.md]: "appboy_api_key" vs. "braze_api_key"

Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/push_story.md]: fixed

File: _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md [hunk 2]
```diff
@@ -61,10 +61,10 @@ In some cases (e.g., on session start), the in-app message may be triggered and
...
 
 ```swift
-Appboy.startWithApiKey("YOUR-API-KEY",
-                       inApplication:application,
-                       withLaunchOptions:launchOptions,
-                       withAppboyOptions:[ ABKInAppMessageControllerDelegateKey : self ])
+Appboy.start(withApiKey: "YOUR-API-KEY",
+                 in:application,
+                 withLaunchOptions:launchOptions,
+                 withAppboyOptions:[ ABKInAppMessageControllerDelegateKey : true ])
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md]: Why did we change this to `true`?

Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md]: great catch - fixed

**Comment by KellieHawks** [comment]
@cla-bot check

**Comment by Bucimis** [comment]
@KellieHawks ready, please merge

**Comment by zzhaobraze** [comment]
@cla-bot check

**Comment by zzhaobraze** [comment]
@cla-bot check

**Pull request merged**